### PR TITLE
Refactored read operator message to use a list new struct for data

### PR DESCRIPTION
**Pull request by MSysum**
Refactored read operator message to use a list of OPMSG_DATA, OPMSG class is a list of OPMSG_DATA. Refactored cnc_rdopmsg3   to work with 16I.  the function returns the list of meesages and how many messages are returned. numbeRead is the number of actual messages returned. If the filed datano is -1 then there is no returned message.  Please verify against a 31i.

### CODE REVIEW

**Comment by peteoxenham** [comment]
I'll review this tonight!

**Comment by MSysum** [comment]
Adding a decorator to the focus structs to make manipulation of the _fields_ easier.  This allows ```  _fields_ = [('name',ctypes_char * 255)]``` to become ``` name: ctypes.c_char * 255```

File: examples/focas_read_operator_messege.py
Comment (peteoxenham) [inline examples/focas_read_operator_messege.py:1]: Please correct spelling on file name :)

**Comment by peteoxenham** [comment]
Closing, all these changes are in PR #4

**Pull request closed without merging**